test/gui/shared/scripts/helpers/SyncHelper.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+
+# File syncing in client has the following status
+SYNC_STATUS = {
+    'SYNC': 'STATUS:SYNC',  # sync in progress
+    'OK': 'STATUS:OK',  # sync completed
+    'ERROR': 'STATUS:ERROR',  # sync error
+    'IGNORE': 'STATUS:IGNORE',  # sync ignored
+    'NOP': 'STATUS:NOP',  # not in sync yet
+    'REGISTER': 'REGISTER_PATH',
+    'UNREGISTER': 'UNREGISTER_PATH',
+    'UPDATE': 'UPDATE_VIEW',
+}
+
+SYNC_PATTERNS = {
+    # default sync patterns for the initial sync (after adding account)
+    # the pattern can be of TWO types depending on the available resources (files/folders)
+    'initial': [
+        # when syncing empty account (hidden files are ignored)
+        [SYNC_STATUS['UPDATE'], SYNC_STATUS['OK']],
+        # when syncing an account that has some files/folders
+        [SYNC_STATUS['SYNC'], SYNC_STATUS['OK']],
+    ],
+    'synced': [SYNC_STATUS['SYNC'], SYNC_STATUS['OK']],
```

Comment:
why do both have the same sync pattern?

Reply:
We could use get the pattern using index like `['initial'][1]` but I thought that it would be better to make a separate pattern for matching synced status only. Using index might cause problem in the future